core/storage/user/gae_models.py

```diff
@@ -90,6 +90,9 @@ class UserSettingsModel(base_models.BaseModel):
     preferred_audio_language_code = datastore_services.StringProperty(
         default=None, choices=[
             language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES])
+    # Text language preference used for text translations.
```

Comment:
Let's make it clearer that this is the preference when submitting translations, e.g.

```suggestion
    # Language preference when submitting text translations in the contributor dashboard.
```

Reply:
I have updated this.